src/docs/testing/index.md

```diff
@@ -83,6 +83,7 @@ In addition, you must add the following block to your `pubspec.yaml`:
 
 ```yaml
 dev_dependencies:
+  test:
```

Comment:
Change this to `test: ^1.5.1` and place it after the `flutter_test` entry, like this:

```yaml
dev_dependencies:
  flutter_test:
    sdk: flutter
  test: ^1.5.1
```

Reply:
Done!